libbeat/dashboards/import_dashboards.go

```diff
@@ -559,6 +561,9 @@ func (imp Importer) ImportArchive() error {
 		archive = imp.cl.opt.File
 	} else if imp.cl.opt.Url != "" {
 		// it's an URL
+		if imp.cl.opt.Snapshot {
```

Comment:
I think you wanted here to be able to pass either an `-url` or the `-snapshot` option, not both in the same time.